### ISSUE TITLE
my collab notebook crashes when running code

### ISSUE THREAD

**Issue by zakariaelh** [seanghay/uvr#4]
When running a separate.py on an mp3 of 20 min, my collab notebook crashes due to RAM usage (12GB). Are there some parameters I can tweak to reduce the ram usage?

**Comment by seanghay** [comment]
@zakariaelh you can chunk audio into multiple smaller audio and merge it back once it's done.

**Comment by zakariaelh** [comment]
Yeah, that's what I ended up doing. Thank you! I haven't used the GUI version as i was looking a function to incorporate in my flow, so what you built is perfect. Btw, are there some parameters that one can tweak to improve the quality of the audio? Some of the vocals still include clapping in the background

**Comment by seanghay** [comment]
> Yeah, that's what I ended up doing. Thank you! I haven't used the GUI version as i was looking a function to incorporate in my flow, so what you built is perfect. Btw, are there some parameters that one can tweak to improve the quality of the audio? Some of the vocals still include clapping in the background 

Maybe you should try using different models. Try [3_HP-Vocal-UVR.pth](https://huggingface.co/fastrolling/uvr/resolve/main/Main_Models/3_HP-Vocal-UVR.pth?download=true)

**Comment by zakariaelh** [comment]
Excellent, thanks a lot!
